src/devtools/client/debugger/src/components/TestInfo/TestSteps.tsx

```diff
@@ -184,6 +181,29 @@ function TestStepItem({
   const onJumpToAfter = () => {
     dispatch(seekToTime(startTime + adjustedDuration - 1));
   };
+  const onGoToLocation = async () => {
+    const { point } = annotations[index];
+
+    const frame = await (async point => {
```

Comment:
Assuming this is Good Enough for now but noting that we don't need to wrap this with async and this `app.sendMessage` should be replaced with either the underlying `sendMessage` command from the protocol or use the new ReplayClient class.